components/home/Home.bs

```diff
@@ -20,11 +21,80 @@ sub init()
 
     m.homeRows = m.top.findNode("homeRows")
     m.homeRows.focusBitmapBlendColor = chainLookupReturn(m.global.session, "user.settings.colorCursor", ColorPalette.HIGHLIGHT)
+
+    ' Create random background task
+    m.randomBackgroundTask = createObject("roSGNode", "RandomBackgroundTask")
+    m.randomBackgroundTask.observeField("backdropURL", "onBackdropURLChanged")
+
+    ' Set up backdrop overlay with configurable opacity
+    m.backdropOverlay = m.top.findNode("backdropOverlay")
+    overlayOpacity = chainLookupReturn(m.global.session, "user.settings.backgroundOverlayOpacity", 0.94)
+    if isValid(m.backdropOverlay)
+        m.backdropOverlay.opacity = overlayOpacity
+    end if
+
+    ' Only set up random backgrounds if the feature is enabled (default to true)
+    randomBackgroundsEnabled = chainLookupReturn(m.global.session, "user.settings.enableRandomBackgrounds", true)
```

Comment:
This entire thing rests on this setting, but you provided users no method to enable it.

Additionally, users have no means to set any of the settings referenced.